### PR TITLE
quick staging fixes

### PR DESCRIPTION
**Pull request by MaxIsJoe**
CL: [Fix] Fixed some sounds not playing on clients
CL: [Fix] Fixed a few instances of the laydown behavior attempting to call server code from the client.
CL: [Fix] Reduced the size of text in chat templates. People will no longer yell now.

### CODE REVIEW

**Review by Bod9001 — DISMISSED** [review]

**Comment by MaxIsJoe** [comment]
wait for a second, still trying to fix chat sizes

File: UnityProject/Assets/Scripts/Strings/ChatTemplates.cs [hunk 1]
```diff
@@ -5,12 +5,12 @@ namespace Strings
 {
 	public static class ChatTemplates
 	{
-		public const int SmallText = -2;
-		public const int NormalText = 1;
-		public const int LargeText = 8;
-		public const int VeryLargeText = 16;
-		public const int ExtremelyLargeText = 26;
-		public const int AnnoyingText = 48;
+		public const int SmallText = 12;
```
Comment (corp-0) [inline UnityProject/Assets/Scripts/Strings/ChatTemplates.cs]: if I understand this correctly, size used to be ``-2`` but now it is ``12 + FONTSCALE``. So it still won't be the same it used to be. Are you sure you don't want to multiply instead?

Comment (MaxIsJoe) [reply UnityProject/Assets/Scripts/Strings/ChatTemplates.cs]: ignore this change

**Comment by corp-0** [comment]
speed merging is my passion

**Comment by MaxIsJoe** [comment]
am SPEEDDDD

**Pull request merged**